src/View.php

```diff
@@ -790,6 +790,8 @@ public function renderToJsonArr(bool $forceReturn = true, $region = null): array
     public function getHtml()
     {
         if (isset($_GET['__atk_reload']) && $_GET['__atk_reload'] === $this->name) {
+            // prevent from looping if this get render again, i.e. part of a Modal View.
+            $_GET['__atk_reload'] = null;
```

Comment:
_GET, _SET should be always a string by definition and we may rely on this property - fix otherwise ok, but unset it with `unset()` instead

Reply:
also, can we test this using one Behat test?

Reply:
why do you mention `Modal View`, is Modal specific only?

Reply:
As of now - only Modals are render in app-terminate() if type === 'application/json'. So this should only apply to Modal basically. 
I was not sure if it was better to clear the $_GET argument in App:getRenderedModals() instead, because this is where the issue originated from. wdyt? I have done it in View::getHtml() just in case another use case come up but I doubt it.

Reply:
> I was not sure if it was better to clear the $_GET argument in App:getRenderedModals() instead, because this is where the issue originated from. wdyt? I have done it in View::getHtml() just in case another use case come up but I doubt it.

Very good point, it seems like related with App specific impl. (use of terminateJson() method in View), so I think moving it there is much cleaner. I moved it and fixed unset.

At least one Behat test should test this.

Reply:
Behat test is added.